### PR TITLE
#928 change the logic so non-event-based invitations arent sent to us…

### PR DESCRIPTION
**Pull request by tevtonez**
…ers w passwords

### CODE REVIEW

File: openCurrents/templates/layouts/admin.html [hunk 1]
```diff
@@ -51,6 +51,8 @@ <h6 class="half-margin-bottom">
       {% endblock heading-left %}
     </div>
 
+    aaaa: {{org.no_info}}
```
Comment (nickolashe) [inline openCurrents/templates/layouts/admin.html]: @tevtonez what's the purpose of this?

Comment (tevtonez) [reply openCurrents/templates/layouts/admin.html]: oh, my apologies, it's left from debugging stage. Removing it.

**Comment by nickolashe** [comment]
@dannypernik any thoughts?

**Comment by nickolashe** [comment]
@tevtonez is this ready to merge or is the status incorrect?

**Comment by nickolashe** [comment]
@tevtonez could you comment on the status of this PR - is this ready to be merged?

**Comment by tevtonez** [comment]
@nickolashe yes, it's ready to be merged

**Pull request merged**